aim/web/ui/src/components/ScatterPlot/styles.scss

```diff
@@ -10,6 +10,13 @@
   min-width: 100px;
```

Comment:
same issue with px

Reply:
We need to decide with UI team when exactly we use `px` or `toRem(px)`
@roubkar  what do you think about this ?

Reply:
here I think `px` is good